### PR TITLE
Fix(task-details):Fixed Comments Reload

### PR DESCRIPTION
**Pull request by Aditya3034**
### Functionality:
Fixed reload issue in task-details comments.

### Solution:
Changes in task-details component.

### Risk level:
- [ ] high 
- [X] medium
- [ ] low

### How to test:
1.Open emulator
2.comment on any task from my-dashboard.
3.comment will be added without reload.

### CODE REVIEW

File: src/app/body/task-details/task-details.component.ts [hunk 1]
```diff
@@ -267,7 +267,7 @@ export class TaskDetailsComponent implements OnInit {
     }
 
   async addComment() {
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:269]: remove async

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the fix

**Review by Pendyala-thanvi — APPROVED** [review]

**Review by sanjaykrishna1203 — APPROVED** [review]
Approved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**